src/components/reactTasks/comment.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+export default class Comment extends React.Component{
+
+  constructor(){
+    super()
+    this.state = {comments: ['Brilliant!', 'Terrible']}
+  }
+
+
+handleChange(event) {
+    this.setState({value: event.state.value});
+  }
+
+
+  handleClick(){
+    if(this.state.value.length > 0){
+    this.state({comments: _.concat(this.state.comments, this.state.value)})
+    this.state.value = ''
```

Comment:
This works, but we don't use `this.state.whatever = 1` type thing to set state, we use `setState` as you have done before. this is because `this.state.whatever = 1` mutates state (https://medium.com/@fknussel/arrays-objects-and-mutations-6b23348b54aa)